js/graphql.md

```diff
@@ -0,0 +1,1374 @@
+---
+---
+# The GraphQL Transform
+
+The GraphQL Transform is a library that simplifies the process of designing GraphQL APIs.
+In short, it allows a developer to design application backends with a simple `schema.graphql`
+which is then *transformed* into a AWS CloudFormation that implements the data model
+defined in `schema.graphql`. For example you might create the backend for a blog like this:
```

Comment:
...defined in a `schema.graphql` file with annotations such as `@model` or `@connection`.

Reply:
@mikeparisstuff I've been calling these "annotations" recently as it's friendly nomenclature for developers unfamiliar with GraphQL, but happy to call these "directives" if you think that's better.